packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -1049,18 +1034,16 @@ describe( 'Creating Entities (eg: Posts, Pages)', () => {
 				name: /Search results for.*/,
 			} )
 		).getAllByRole( 'option' );
-		const createButton = Array.from( searchResultElements ).filter(
-			( result ) => result.innerHTML.includes( 'Create:' )
-		)[ 0 ];
+		const createButton = searchResultElements.find( ( result ) =>
+			result.innerHTML.includes( 'Create:' )
+		);
 
 		// Step down into the search results, highlighting the first result item.
 		triggerArrowDown( searchInput );
 
-		createButton.focus();
-		await user.keyboard( '[Enter]' );
+		await user.click( createButton );
```

Comment:
The description for this test mentions creating entities _via keyboard_ but here we're using a mouse event. As per your PR description it's clear to me why but it may be worth a comment so devs coming here in the future easily understand, too. What do you think?

Reply:
Oh yes, this should be fixed. But when I tried, it becomes a complex task. Let's do it in another PR: this one was supposed to be a search-and-replace easy fix.

> There's one occurrence of searchInput.focus() left on line 754, shall we use the opportunity to wrap it in act?

Yes, but later, for the same reasons.